examples/redirectRuleExample.js

```diff
@@ -58,11 +58,11 @@ async function redirectRuleExample(user, context, callback) {
   }
 
   // Some kind of context check occurred to determine if a redirect should happen.
-  if (ruleUtils.canRedirect && !userUtils.setAppMeta("is_verified")) {
+  if (ruleUtils.canRedirect && !userUtils.getAppMeta("is_verified")) {
     try {
       // This method automatically creates a session token.
-      // To add data to this token, use ruleUtils.createSessionToken and pass as second param below.
-      // To omit the session token, pass false as second param below.
+      // To add data to this token, use ruleUtils.createSessionToken and pass { sessionToken: yourSessionToken } as second param below.
```

Comment:
I think the API must have changed at some point where these values could be passed directly as the second parameter in the `doRedirect` method. Now they are more flexibly embedded within an `options` object. So just correcting the example to follow the new API.